pubsub_nats_options.go

```diff
@@ -70,8 +68,15 @@ func natsOptClient(client natsClient) NATSOption {
 var ackMessage = []byte("ack")
 
 type natsSubscribeConfig struct {
-	queueGroup string
-	replier    func(msg *nats.Msg) []byte
+	queueGroup   string
+	replyTimeout time.Duration
+}
+
+func defaultSubscribeConfig() natsSubscribeConfig {
+	return natsSubscribeConfig{
+		queueGroup:   "",
```

Comment:
not very useful initializer

Reply:
This is necessary because we don't to force the client to using `QueueSubscribe` mode by default - see https://github.com/aporeto-inc/bahamut/blob/master/pubsub_nats.go#L156

If they want to use `QueueSubscribe`, they can provide the `NATSOptSubscribeQueue` option.

Reply:
Yes. But initializing a string to "" is not useful

Reply:
That's the default value for strings, even if I leave it out, it will be set to the empty string.

Do you want to leave it out so it makes it obvious to the reader that we intentionally don't initialize this value?